libsecret.go

```diff
@@ -72,7 +97,7 @@ func (k *secretsKeyring) openSecrets() error {
 	path := libsecret.DBusPath + "/collection/" + k.name
 
 	for _, collection := range collections {
-		if string(collection.Path()) == path {
+		if decodeKeyringString(string(collection.Path())) == path || string(collection.Path()) == path {
```

Comment:
if there are no dashes in the path, will `decodeKeyringString(string(collection.Path()))` be the same as `string(collection.Path())`? If so, I guess no need for that or condition?

Reply:
yes, only decodeKeyringString is enough.
i'll change it
because this source is only for linux.